src/components/common/Input/Input.tsx

```diff
@@ -0,0 +1,23 @@
+import { ComponentProps } from 'react';
+import './Input.scss';
+import _ from 'lodash';
+
+interface InputProps extends ComponentProps<'input'> {
+  label?: string;
+}
+
+export const Input = (props: InputProps) => {
+  const { label, ...inputProps } = props;
+  const inputId = inputProps.id || _.uniqueId('input_');
```

Comment:
I have a feeling `_.uniqueId('input_')` will be regenerated on every render of this component. `id` should be a mandatory prop.

Reply:
I moved it to the constructor as discussed... would that be enough? or do you think it's better to mark it as required?